packages/storage/amplify_storage_s3/CHANGELOG.md

```diff
@@ -1,3 +1,28 @@
+## 1.0.0
+
+We are thrilled to release version 1.0 of the Amplify Flutter libraries!
+
+This release introduces a full rewrite of the Amplify libraries in Dart, allowing you to build AWS-powered
+applications on every platform Flutter supports (iOS, Android, Web, macOS, Windows, and Linux).
+
+If you've been using v0 in production, you can now safely migrate to v1. Check out our 
+[migration guide](https://docs.amplify.aws/lib/project-setup/upgrade-guide/q/platform/flutter/) for 
+more information.
+
+As always, you can find us on [GitHub](https://github.com/aws-amplify/amplify-flutter/) and 
+[Discord](https://discord.gg/jWVbPfC) to answer any questions.
+
+### New Features
+
+* In-flight download and upload operations are now fully controllable.
+    * NOTE: Downloading and uploading in the background is not yet supported.
+* The `list` API now supports pagination.
+* The new APIs `uploadData` and `downloadData` provide flexibility when dealing with data stored in memory.
+    * `uploadData` supports uploading from various data formats such as raw bytes, JSON, strings, Data URLs, and more.
+* The new API `getProperties` allows retrieving S3 object metadata without downloading the object. 
+* New `copy`, `move`, and `removeMany` APIs now support better control of your files.
+* Prefix customization is now supported.
+
```

Comment:
can we also add here a section to link to all the bug fixes we are addressing with this release? I think we had a label we were using already.